ios/Runner/Assets.xcassets/LaunchImage.imageset/Contents.json

```diff
@@ -1,17 +1,17 @@
 {
   "images" : [
     {
-      "filename" : "LaunchImage@1x.png",
+      "filename" : "AppIcon@2x.png",
```

Comment:
Is this an accident? 2x in the name for 1x and 3x in the name for 2x. I could be confused...

Reply:
I just put random images in there, I should probably fix those